src/Plugin.php

```diff
@@ -190,9 +193,17 @@ public static function oembed_fetch_url($provider, $url, $args) {
   }
 
   /**
+   * @implements xmlrpc_methods
+   */
+  public static function xmlrpc_methods($methods) {
+    unset($methods['pingback.ping']);
```

Comment:
What about the second pingback method?
https://github.com/WordPress/WordPress/blob/339d8384755f0355c076fb7cf2e299cce79bd453/wp-includes/class-wp-xmlrpc-server.php#L149